app/grants/templates/grants/components/card.html

```diff
@@ -0,0 +1,229 @@
+            </button>
+          </template>
+        </div>
+    </div>
+    <div class="grant-item__footer">
+      <template v-if="1">
+          <div class="clr mb-3">
+            <div class="match__round pb-1 pt-2 mb-2">
+              <h2 class="title col-8 font-caption g-bold">
+                  CLR MATCH ROUND 6
+              </h2>
+            </div>
+            <div class="col-6 float-right px-0">
+              <h2 class="font-title font-weight-bold amount pt-1 mb-2">
+                [[ grant.amount_received_in_round ]] DAI
+              </h2>
+              <p class="amount mb-2 font-caption font-weight-semibold">
+              raised from [[ grant.positive_round_contributor_count ]] contributors
+              </p>
+            </div>
+            <div class="col-6">
+                <template v-if="grant.is_clr_eligible">
+                  <h2 class="font-title font-weight-bold amount pt-1 mb-2">
+                    {# format curve.0.1|floatformat:0|intcomma|default:0 #}
+                    [[ get_clr_prediction(0, 1) || 0 ]] DAI
+                  </h2>
+                  <p class="amount mb-2 font-caption font-weight-semibold">
+                    {#% if grant.id in round_5_5_grants %#}
+                    {# [[ grant_in_round_5_5 ? 'Current Round 5.5 Match' : 'Final CLR Match']]  #}
+                    Final CLR Match
+                  </p>
+                </template>
+                <template v-else>
+                  <h2 class="font-title font-weight-bold amount pt-1 mb-2">
+                    {# format |floatformat:0|intcomma #}
+                    [[ grant.monthly_amount_subscribed ]] DAI
+                  </h2>
+                  <p class="amount mb-2 font-caption font-weight-semibold">MONTHLY RECURRING</p>
+                </template>
+            </div>
+          </div>
+
+
+          <div class="col-12 pb-3" v-if="grant.is_clr_eligible">
+            {# with short=True #}
+            {% include 'grants/card/clr_estimate.html'  %}
+          </div>
+
+          <h2 class="font-subheader text-center amount pt-4 mb-3" style="color:#069953" v-else>
+            NOT ELIGIBLE FOR CLR
+          </h2>
+
+          <div class="col-12 pb-4">
+            {#% is_team_member grant profile as is_on_team %#}
+            <form class="js-addDetailToCart-form" v-if="!grant.is_on_team">
+              {% include 'grants/components/hidden_inputs.html' %}
+              <button class="btn btn-gc-blue cart-btn large font-body font-weight-semibold mt-1 mr-1">
+                + Add to Cart
+              </button>
+            </form>
+
+            <button class="btn btn-gc-grey cart-btn large font-body font-weight-semibold mt-1 mr-1" v-else>
+              <i class="fa fa-stop" aria-hidden="true"></i> Cannot Fund your Own Grant
+            </button>
+          </div>
+        </template>
+    </div>
+  </a>
+    </div>
+</grant-card>
```

Comment:
^ could you redo the text and the conditions based on what's in `front.html` 
this code seems a lil outdated